libcxx/include/__config

```diff
@@ -805,6 +805,13 @@ typedef __char32_t char32_t;
 // the implementation of a virtual function in an ABI-incompatible way in the first place,
 // since that would be an ABI break anyway. Hence, the lack of ABI tag should not be noticeable.
 //
+// The macro can be applied to record and enum types. When the tagged type is nested in
+// a record this "parent" record needs to have the macro too. Another use case for applying
+// this macro to records and unions is to apply an ABI tag to inline constexpr variables.
+// The macro can be directly applied to an inline constexpr variable. This is useful for the
+// Unicode data tables; these depend on the Unicode version used. Updating the Unicode
+// version may affect the size of the tables.
```

Comment:
This comment should be reworded to be Unicode-agnostic. You can say something like "This can be useful for inline variables that are implementation details which are expected to potentially change in the future".